sympy/functions/elementary/tests/test_complexes.py

```diff
@@ -446,6 +446,17 @@ def test_Abs():
     arg = sqrt(acos(1 - I)*acos(1 + I))
     assert abs(arg) == arg
 
+    # special handling to put Abs in denom
+    assert abs(1/x) == 1/Abs(x)
```

Comment:
```suggestion
    assert abs(1/x) == 1/Abs(x)
    assert abs(2/x**2) == 2/Abs(x**2)
```
Just to confirm that Mul with integers are factored out (are they?) and works for higher orders as well (does it?).